src/routes/tooltips/+page.svelte

```diff
@@ -40,11 +47,14 @@
 	</section>
 	<section class="space-y-8">
 		<h2 class="h2">Open state</h2>
-		<pre class="pre">{`
-import {useState} from 'react';\n
-function Tooltip() {
-	const [isOpen, setIsOpen] = useState(false);
-}`.trim()}</pre>
+		<CodeBlock
+			code={`
+import { useFloating } from '@skeletonlabs/floating-ui-svelte';
+
+// TBD`.trim()}
```

Comment:
I'd suggest moving the trim into the Code Block itself

Reply:
Done!